### PR TITLE
Add work around for non-existant INTERFACE_INCLUDE_DIRECTORIES from imported targets

### PR DESCRIPTION
**Pull request by white238**
This is currently blocking a user.  They are testing it to make sure the hard CMake error goes away.

### CODE REVIEW

File: src/cmake/thirdparty/SetupAxomThirdParty.cmake [hunk 1]
```diff
@@ -222,3 +222,37 @@ foreach(dep ${TPL_DEPS})
...
+        get_target_property(_dirs ${_target} INTERFACE_INCLUDE_DIRECTORIES)
+        set(_existing_dirs)
+        foreach(_dir ${_dirs})
+            if (EXISTS "${_dir}")
+                list(APPEND _existing_dirs "${_dir}")
+            endif()
+        endforeach()
+        if (_existing_dirs)
+            set_target_properties(${_target} PROPERTIES
+                                  INTERFACE_INCLUDE_DIRECTORIES "${_existing_dirs}" )
```
Comment (kennyweiss) [inline src/cmake/thirdparty/SetupAxomThirdParty.cmake:255]: Does this overwrite the existing value of `INTERFACE_INCLUDE_DIRECTORIES`?

**Review by kennyweiss — APPROVED** [review]
Thanks @white238

File: src/cmake/thirdparty/SetupAxomThirdParty.cmake (hunk 1 above)
Comment (kennyweiss) [reply src/cmake/thirdparty/SetupAxomThirdParty.cmake:255]: (My assumption is that it does)

Comment (white238) [reply src/cmake/thirdparty/SetupAxomThirdParty.cmake:255]: yes.. it removes all non-existing directories and overwites the previous value

**Comment by white238** [comment]
closed in preference to #550

**Pull request closed without merging**